source/features/avoid-accidental-submissions.tsx

```diff
@@ -10,11 +10,11 @@ function addQuickSubmit(): void {
 
 function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLInputElement>): void {
 	if (
-		event.key !== 'Enter' ||
-		event.ctrlKey ||
-		event.metaKey ||
-		event.isComposing || // #4323
-		select.exists('.suggester', event.delegateTarget.form!) // GitHub’s autocomplete dropdown
+		event.key !== 'Enter'
+		|| event.ctrlKey
```

Comment:
I wish we could enforce this to be:
```suggestion
			|| event.ctrlKey
```